Platformers/PlatformerWithMoment/Project.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project>
+	<!------------------------------APPLICATION SETTINGS---------------------------->
+	
+	<app title="Moment" file="PlatformerWithMoment" main="Main" version="0.0.1" company="DerivativesStudio" />
+	
+	<!--The flixel preloader is not accurate in Chrome. You can use it regularly if you embed the swf into a html file
+	<!--or you can set the actual size of your file manually at "FlxPreloaderBase-onUpdate-bytesTotal"!-->
+	<app preloader="flixel.system.FlxPreloader" />
+	
+	<!--Minimum without FLX_NO_GAMEPAD: 11.8, without FLX_NO_NATIVE_CURSOR: 11.2 -->
+	<set name="SWF_VERSION" value="11.8" />
+	
+	<!--------------------------------WINDOW SETTINGS------------------------------->
+	
+	<!--These window settings apply to all targets-->
+	<window width="1024" height="768" fps="60" background="#000000" hardware="true" vsync="true" />
+	
+	<!--Web-specific-->
+	<window if="web" orientation="portrait"  />
+	
+	<!--Desktop-specific-->
+	<window if="desktop" orientation="landscape" fullscreen="false" resizable="true" />
+	
+	
+	<!--Mobile-specific-->
+	<window if="mobile" orientation="landscape" fullscreen="true" width="0" height="0" />
+	
+	<!--------------------------------PATHS SETTINGS-------------------------------->
+	
+	<set name="BUILD_DIR" value="export" />
+	<classpath name="source" />
+	<assets path="assets" />
+	
+	<!--------------------------------LIBRARIES------------------------------------->
+	<haxelib name="openfl"/>
```

Comment:
The openfl include here should be removed, this already happens in Flixel's `include.xml`.